### ISSUE TITLE
-Wformat in drivers/net/ethernet/ti/netcp_core.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#1827]
```
drivers/net/ethernet/ti/netcp_core.c:2085:4: warning: format specifies type 'long' but the argument has type 'int' [-Wformat]
                        MAX_SKB_FRAGS);
                        ^~~~~~~~~~~~~
include/linux/dev_printk.h:144:65: note: expanded from macro 'dev_err'
        dev_printk_index_wrap(_dev_err, KERN_ERR, dev, dev_fmt(fmt), ##__VA_ARGS__)
                                                               ~~~     ^~~~~~~~~~~
include/linux/dev_printk.h:110:23: note: expanded from macro 'dev_printk_index_wrap'
                _p_func(dev, fmt, ##__VA_ARGS__);                       \
                             ~~~    ^~~~~~~~~~~
include/linux/skbuff.h:352:23: note: expanded from macro 'MAX_SKB_FRAGS'
#define MAX_SKB_FRAGS CONFIG_MAX_SKB_FRAGS
                      ^~~~~~~~~~~~~~~~~~~~
./include/generated/autoconf.h:11789:30: note: expanded from macro 'CONFIG_MAX_SKB_FRAGS'
#define CONFIG_MAX_SKB_FRAGS 17
                             ^~
1 warning generated.
```

Patch submitted: https://lore.kernel.org/20230329-net-ethernet-ti-wformat-v1-1-83d0f799b553@kernel.org/

**Comment by nathanchance** [comment]
Accepted: https://git.kernel.org/netdev/net-next/c/3292004c90c8

**Comment by nathanchance** [comment]
Fixed in next-20230403.
